data/human/hails.txt

```diff
@@ -2195,6 +2195,362 @@ phrase "friendly civilian"
 		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
 		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`
 
+phrase "friendly civilian"
+	word
+		"If you want to "
+		"To "
+	word
+		"make"
+		"turn"
```

Comment:
```suggestion
		"earn"
```

earn gives more variety, since you use "turnover" below. Gotta change the replacements too, if going this route.

You could also just add it to the list.

```suggestion
		"turn"
		"earn"
```